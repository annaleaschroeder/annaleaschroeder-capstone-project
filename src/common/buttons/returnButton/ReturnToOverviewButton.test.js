import React from 'react'
import ReturnToOverviewButton from './ReturnToOverviewButton'
import { render } from '@testing-library/react'

describe('ReturnToOverviewButton', () => {
  it('renders correctly', () => {
    const { container } = render(<ReturnToOverviewButton />)
    expect(container).toMatchSnapshot()
  })
})
