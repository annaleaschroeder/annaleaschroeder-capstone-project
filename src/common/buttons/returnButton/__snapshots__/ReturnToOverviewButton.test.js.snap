// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReturnToOverviewButton renders correctly 1`] = `
<div>
  <div
    class="ReturnToOverviewButton__ButtonContainer-sc-12mn0g6-0 bAjjFv"
  >
    <a
      class="ReturnToOverviewButton__ReturnRoute-sc-12mn0g6-1 saeg"
      href="/"
    >
      <button
        class="ReturnToOverviewButton__ReturnButton-sc-12mn0g6-2 cLMUAI"
      >
        <svg
          class="ReturnToOverviewButton__ReturnImage-sc-12mn0g6-3 dRmaoe"
          height="40"
          viewBox="0 0 24 24"
          width="40"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M0 0h24v24H0z"
            fill="none"
          />
          <path
            d="M20 11H7.83l5.59-5.59L12 4l-8 8 8 8 1.41-1.41L7.83 13H20v-2z"
          />
        </svg>
      </button>
    </a>
  </div>
</div>
`;
